paper_trail.gemspec

```diff
@@ -19,9 +19,13 @@ has been destroyed.
   s.email = "jared@jaredbeck.com"
   s.license = "MIT"
 
-  s.files = `git ls-files -z`.split("\x0").select { |f|
-    f.match(%r{^(Gemfile|LICENSE|lib/|paper_trail.gemspec)})
-  }
+  s.files = Dir.glob([
+    "CHANGELOG.md",
+    "doc/warning_about_not_setting_whodunnit.md",
+    "lib/**/*",
+    "LICENSE",
+    "README.md"
+  ])
```

Comment:
Hi @jaredbeck 👋🏻 

> I think the main point here is to stop using git to generate our file list. That's fine with me.
>
> To keep gem size down, I'd prefer not to begin shipping documentation (changelog, readme, /doc). Adding documentation would double the size of the gem. It's my personal experience that people will read the documentation online. If I'm wrong, let's discuss this change (the introduction of docs) in a separate PR please, and focus here on the Gemfile and gemspec.
>
> Without a clear answer from the specification, I'd prefer to simply not change this. That is, let's not include directories.
>
> Finally, regarding the removal of the .gemspec file, I couldn't find documenation, but it seems conventional. Neither ActiveRecord nor Devise (simply the first two gems I looked at) ship their .gemspec. Less surprising to me, neither ships a Gemfile. So, I think the removal of both files is fine.

Okay, that's fair enough! 👍🏻 
Here's something that you'd probably like to go with then:


```suggestion
  s.files = Dir["lib/**/*", "LICENSE"].reject { |f| File.directory?(f) }
```

This ensures no directories are included, git is dropped, and gemspec and Gemfile are dropped as well!
I hope we're on the same page with this^?